activators/lex/README.md

```diff
@@ -0,0 +1,96 @@
+<p style="text-align: center">
+    <img src="https://www.routeoneconnect.com/wp-content/uploads/2020/03/icon-amazon-lex.png" height="128" width="128" alt="Lex logo"/>
+</p>
+
+<h1 style="text-align: center">Lex NLU activator</h1>
+
+Allows using [Amazon Lex](https://aws.amazon.com/lex/) NLU engine as a states activator in JAICF.
+
+_Built on top of [Amazon Lex API Client for Java](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/)._
```

Comment:
We're actually using only lexruntimev2

https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/lexruntimev2/AmazonLexRuntimeV2.html